lib/begin.go

```diff
@@ -179,6 +190,9 @@ func (a *ACBuild) beginFromImage(start string, insecure bool) error {
 
 	err = reg.Fetch(app.Name, labels, 0, false)
 	if err != nil {
+		if _, ok := err.(*url.Error); ok {
+			return fmt.Errorf("unknown host when fetching image, local file paths must start with '/' or '.'")
```

Comment:
Weird, what is the content of the error?

An alternative approach is to actually do a stat and see if there is a file they were trying to reference.. I don't know if I like that though.